Makefile

```diff
@@ -22,7 +22,7 @@ ENVTEST_ARCH ?= amd64
 all: manager
 
 # Download the envtest binaries to testbin
-ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
+ENVTEST_ASSETS_DIR=$(shell pwd)/build/testbin
```

Comment:
Is `testbin/` still used for something (as it's still included in `.gitignore`)

Reply:
Not at all, I have pushed a new version removing it from the `.gitignore`. Thanks for pointing it out.